lib/carbon/database.py

```diff
@@ -51,7 +51,7 @@ def setMetadata(self, metric, key, value):
     "Modify metric metadata."
     raise NotImplementedError()
 
-  def getFilesystemPath(self, metric):
+  def getFilesystemPath(self, metric, tag_hash_filenames=None):
```

Comment:
Does it need to be exposed to the Database API ? Looks like a whisper specific thing to me.

Reply:
It's used by Ceres also, if it's a concern I could switch both Whisper and Ceres to use an internal _ version that takes the additional parameter

Reply:
I'll do that, it'll be cleaner that way